src/es/mahnwalatino/src/eu/kanade/tachiyomi/extension/es/manhwalatino/ManhwaLatinoSiteParser.kt

```diff
@@ -261,15 +261,20 @@ class ManhwaLatinoSiteParser(
      */
     fun getPageListParse(response: Response): List<Page> {
         val document = response.asJsoup()
-        val scripUrl = document.select(MLConstants.pageListParseSelector).attr("src")
-        val script = client.newCall(GET(scripUrl, headers)).execute().asJsoup().text()
+        val script = document.select(MLConstants.pageListParseSelector)
 
-        val password = script
+        val scriptData: String = if (script.hasAttr("src")) {
+            client.newCall(GET(script.attr("src"), headers)).execute().asJsoup().text()
```

Comment:
Does the script returns an HTML somehow? Can't you just use this instead?

```suggestion
            client.newCall(GET(script.attr("src"), headers)).execute().body.string()
```